src/Fls.Results/Fls.Results.csproj

```diff
@@ -2,6 +2,21 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-  </PropertyGroup>
 
+    <IncludeSymbols>true</IncludeSymbols>
+    <IncludeSource>true</IncludeSource>
+    <PackageId>FlsResults</PackageId>
+    <Title>Project FlsResults</Title>
+    <Version>$(APPVEYOR_BUILD_VERSION)</Version>
+    <Authors>FirstLine Net Core Practice Team</Authors>
+    <Company>FirstLine Software</Company>
+    <license>https://raw.githubusercontent.com/vkamiansky/results/master/LICENSE</license>
+    <PackageProjectUrl>https://github.com/vkamiansky/results</PackageProjectUrl>
+    <PackageTags>study;practice;</PackageTags>
+    <RepositoryUrl>https://github.com/vkamiansky/results.git</RepositoryUrl>
+    <Description>A C# framework for building railroad-oriented code with results.</Description>
```

Comment:
A C# framework for building railroad-oriented code with generic results.

Reply:
Here is the same text as you offered